.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,35 @@
+name: Feature Request
```

Comment:
Detection Rules has evolved over time into a multi-faceted repository with the following "features" per say:

- Stored Detection Rules (rules, validation, testing, etc.)
- Detections-as-Code (DaC)
- Hunting Library

Not actionable item now, but later on if we continue to add features, maybe we update docs to acknowledge that the repo is not just rules anymore.